frame/treasury/src/benchmarking.rs

```diff
@@ -107,20 +107,70 @@ fn create_approved_proposals<T: Trait>(n: u32) -> Result<(), &'static str> {
 	Ok(())
 }
 
+// Create bounties that are approved for use in `on_initialize`.
+fn create_approved_bounties<T: Trait>(n: u32) -> Result<(), &'static str> {
+	for _ in 0 .. n {
+		let (caller, _curator, _fee, value, reason) = setup_bounty::<T>(MAX_BYTES);
```

Comment:
same here, all bounties are using same accounts, isn't this an issue when you want to benchmark spend_funds ?